### PR TITLE
Pba 3908 upgrade skin to use react-native 0.26.2

### PR DESCRIPTION
**Pull request by Matthew-C**

### CODE REVIEW

**Comment by lental** [comment]
This looks pretty close to what I expected.

Will this change much with the Sample App's build.gradle?

**Comment by Matthew-C** [comment]
Nope, you can checkout android sample app PBA-3908 to take a look.

**Comment by lental** [comment]
does this fix PBA-3412?

File: sdk/android/ooyalaSkinSDK/build.gradle [hunk 1]
```diff
@@ -25,8 +25,37 @@ android {
     }
 }
 
+repositories {
+    jcenter()
+    flatDir {
+        dirs 'libs'
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/build.gradle:31]: any reason you need to flatDir?

Comment (Matthew-C) [inline sdk/android/ooyalaSkinSDK/build.gradle:31]: this is required to process aar.

**Pull request merged**